Content.Client/Weapons/Melee/UI/MeleeSpeechBoundUserInterface.cs

```diff
@@ -0,0 +1,58 @@
+using Robust.Client.GameObjects;
+using Content.Shared.Weapons.Melee;
+
+namespace Content.Client.Weapons.Melee.UI
```

Comment:
```suggestion
namespace Content.Client.Weapons.Melee.UI;
```

Convert this to a declared namespace. It allows you to remove the extra {} and tab everything back by one. Makes the code look neater.